src/I18n/Keys.elm

```diff
@@ -54,6 +56,18 @@ type Key
       --- Story Page
     | WhoSubHeading
     | WhereSubHeading
+      --- SubmitStory Page
+    | SubmitStoryTitle
+    | SubmitStoryP1
```

Comment:
Is it worth moving these into CMS also? Or is that more complicated than it sounds?

Reply:
we'd need to make a new field for meta description on all the pages and that feels like a new feature

Reply:
ok I think I resolved it see code in `Metadata.elm`

I've also updated the copy spreadsheet to reflect this